foundations/html_css/html-foundations/project-recipes.md

```diff
@@ -73,9 +86,9 @@ GitHub allows you to publish web projects directly from a GitHub repository. Doi
 
 There are a couple of ways to go about doing this, but the simplest is this:
 
-- make sure that the main HTML file of your project is called `index.html`. If it is not, you will need to rename it.
-- go to **your GitHub repo** on the web.
-- click on the **Settings** button from the panel at the top.
-- click on **Pages** on the left side bar.
-- change the **Source** from _none_ to _main branch_ and click Save.
-- it may take a few minutes (the GitHub website says up to 10, but we've seen it take up to an hour. Do not add a "theme" to your project, or you may have git conflicts, instead, be patient.) but your project should be accessible over the web from `your-github-username.github.io/your-github-repo-name` (obviously substituting your own details in the link).
+-   make sure that the main HTML file of your project is called `index.html`. If it is not, you will need to rename it.
+-   go to **your GitHub repo** on the web.
+-   click on the **Settings** button from the panel at the top.
+-   click on **Pages** on the left side bar.
+-   change the **Source** from _none_ to _main branch_ and click Save.
+-   it may take a few minutes (the GitHub website says up to 10, but we've seen it take up to an hour. Do not add a "theme" to your project, or you may have git conflicts, instead, be patient.) but your project should be accessible over the web from `your-github-username.github.io/your-github-repo-name` (obviously substituting your own details in the link).
```

Comment:
```suggestion
-   it may take a few minutes (the GitHub website says up to 10, but we've seen it take up to an hour. Do not add a "theme" to your project, or you may have git conflicts, instead, be patient.) but your project should be accessible over the web from `your-github-username.github.io/your-github-repo-name` (obviously substituting your own details in the link).

```
Newline at end of file